test/integration/visitor_can_add_items_to_cart_test.rb

```diff
@@ -33,14 +33,75 @@ class VisitorCanAddItemsToCart < ActionDispatch::IntegrationTest
   end
 
   test "can add multiple items to cart" do
-    skip
+    category_1 = Oil.create(name: "Lard")
+    category_2 = Oil.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_1`. Did you mean `item_2`?<br>Line is too long. [106/80]